client/src/components/Home/index.js

```diff
@@ -249,11 +249,11 @@ export default function Home(props) {
               <p>
                 <a href="#" className="header-link">
```

Comment:
We'll need to replace the `#` here by the `https://docs.rocket.chat/guides/user-guides` (with the quotes) for it to redirect.

Reply:
done thx